ui/pages/destinations/[id]/index.js

```diff
@@ -841,63 +841,31 @@ export default function DestinationDetail() {
                       privilege,
                       selectedResources,
                     }) => {
-                      // don't add grants that already exist
```

Comment:
Since this is no longer calling the API once for each resource, it's less critical to filter out existing grants